website/.vuepress/config.ts

```diff
@@ -18,9 +18,6 @@ import { defineUserConfig } from "vuepress";
 import theme from "./theme.js";
 
 export default defineUserConfig({
-  head: [
-    ['link', { rel: 'icon', href: '/logo.jpg' }]
```

Comment:
Why these lines are removed

Reply:
because the `head` is repeated now, code is here:

https://github.com/bytedance/bitsail/blob/1b5478592a0c6d85a1e370556f3a8f0db6bbfad0/website/.vuepress/config.ts#L20-L40